src/test/groovy/io/sqreen/powerwaf/ByteBufferSerializerTests.groovy

```diff
@@ -308,7 +308,7 @@ class ByteBufferSerializerTests implements PowerwafTrait {
     void 'observes maximum string size'() {
         maxStringSize = 3
         // the size is number of UTF-16 code units
-        def str = "\uFFFD" * 3 + 'x'
+        def str = '\uFFFD' * 3 + 'x'
```

Comment:
People almost never realize that double quotes in Groovy means a `GString` and it is not a `String`, it works 99.999% of the time, but when it fails it's usually a pain in the *** :smile:

Reply:
If I recall correctly, double quotes `"..."` will be interpreted as `GString` only there is interpolation (e.g. contains `${...}`) otherwise it still will be a Java String.